ui/components/MesheryPerformanceComponent.js

```diff
@@ -78,6 +79,8 @@ const styles = (theme) => ({
   },
 });
 
+const prePopulatedOptions = ['15s', '30s', '1m', '3m', '5m', '10m', '30m', '1h', '2.5h', '5h', '10h', '1d']
```

Comment:
These are a lot to options we just want to make users know how we support entering the values.

Reply:
I don't understand you mean. just a review msg ?

Reply:
Hi @PmcFizz , We want the dropdown to just let users know what sort of string or what sort of input they want to give. But here we have given 10 pre-populated options which I don't think is mandatory, I think just 4 options like 10s, 1m, 1h should be sufficient to let the user know that we accept the following input. I guess we can cut out some options just to maintain it as a guide of how you should enter the input in following field.

Reply:
Gents, to get this PR over the line, for now, let's stick with the set of values outlined in the issue - https://github.com/layer5io/meshery/issues/1149#issuecomment-654844521.